dev-tools/cc-test/build.rs

```diff
@@ -156,7 +156,13 @@ fn run_action_if_forked() -> bool {
     true
 }
 
+fn disable_debug_output() {
+    std::env::remove_var("CC_ENABLE_DEBUG_OUTPUT");
```

Comment:
Is there a reason we're mutating the environment here...?

Reply:
That env would break tests for warning/debug output

Reply:
And it is set in the CI, to make debugging CI failure easier.

Reply:
Ah, I see, this is purely testing code. Can you leave a comment about that?

Reply:
Sure, will do